phpcs.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
Do we even need this file?

Reply:
I think this is an incredibly nice QOL addition and I'd vote to keep.

Reply:
It is not strictly necessary but can be a place where project team can easily exclude dirs, customize rules and etc.

Reply:
Definitely include this file. Many IDEs will search for this file when determining what ruleset to lint.

Reply:
It's not covered in the 10up default ruleset, and this file is meant to be extended anyway, but for a theme I wouldn't mind having boilerplate stuff for proper text domain usage and function namespace.

```
<rule ref="WordPress.WP.I18n">
	<properties>
		<property name="text_domain" type="array" value="tenup-scaffold,default"/>
	</properties>
</rule>
<rule ref="WordPress.NamingConventions.PrefixAllGlobals">
	<properties>
		<property name="prefixes" type="array" value="tenup_scaffold"/>
	</properties>
</rule>
```